docs/parameters_desc.md

```diff
@@ -0,0 +1,189 @@
+, <code>alphaMCD = 0.9</code>, for example.
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.mcd.stroppingDelta</code></td>
+  <td><code>0.001</code></td>
+  <td>
+  This is again a <b>MCD-only</b> parameter. Controls
+the stopping condition in the training phase of MCD.
+  </td>
+</tr>
+</table>
+
+
+The following are streaming-specific parameters, and make sense _only_ in
+streaming workloads.
+
+<table class="table">
+<tr><th>Name</th><th>Default</th><th>Meaning</th></tr>
+<tr>
+  <td><code>macrobase.analysis.streaming.inputReservoirSize</code></td>
+  <td><code>10,000</code></td>
+  <td>
+  Controls the number of points sampled from the
+input data stream. Whenever the outlier detection algorithm is trained, the tuples
+that are currently in the input reservoir are used. The larger this parameter,
+the more context the detection algorithms will have in the training phase. (and
+consequently, the longer the training phase will take)
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.streaming.scoreReservoirSize</code></td>
+  <td><code>10,000</code></td>
+  <td>
+  Controls the number of points sampled from the
+scored data stream. Used to estimate the threhold score when `usePercentile`
+(described above) is set to <code>true</code>. The larger this parameter, the more accurate
+the chosen threshold will be. (although more time would be spent to keep
+the reservoir sorted)
+  </td>
+</tr>
+  <td><code>macrobase.analysis.streaming.summaryUpdatePeriod</code></td>
+  <td><code>100,000</code></td>
+  <td>
+  Unused parameter.
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.streaming.modelUpdatePeriod</code></td>
+  <td><code>100,000</code></td>
+  <td>
+  Controls how frequently the model needs to be refreshed.
+Could be a time (in case <code>useRealTimePeriod</code> is set to <code>true</code>) or a tuple count
+(in case <code>useTupleCountPeriod</code> is set to <code>true</code>).
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.streaming.useRealTimePeriod</code></td>
+  <td><code>false</code></td>
+  <td>
+  If true, the model is re-trained at regular periods of time.
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.streaming.useTupleCountPeriod</code></td>
+  <td><code>true</code></td>
+  <td>
+  If true, the model is re-trained at regular tuple counts.
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.streaming.warmupCount</code></td>
+  <td><code>10,000</code></td>
+  <td>
+  Model is initialized after <code>warmupCount</code> tuples filter into the
+system. After <code>warmupCount</code> tuples enter the system, the system retrains the model
+periodically, depending on the settings of <code>modelRefreshPeriod</code>, <code>useRealTimePeriod</code>
+and <code>useTupleCountPeriod</code>.
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.streaming.decayRate</code></td>
+  <td><code>0.01</code></td>
+  <td>
+  Controls how much old tuples are discounted. The lower this parameter
```

Comment:
Exponential decay rate for input and score samples and summary attribute counts. Decay is performed every period, configurable below.